framework/src/modules/chain/transport.js

```diff
@@ -27,6 +27,13 @@ const transactionsModule = require('./transactions');
 const exceptions = global.exceptions;
 const { MAX_SHARED_TRANSACTIONS } = global.constants;
 
+function incrementRelays(packet) {
```

Comment:
Could you make a static function inside the class?

Reply:
@shuse2 which alternative do you propose or what's your point of view here?

Reply:
if it doesn't rely on the state, it should be just a simple function.
We don't use `static` function unless design/UX related point